thunder/tests/opinfos.py

```diff
@@ -1176,6 +1176,27 @@ def _abs_torch(x: torch.Tensor | Number):
 )
 elementwise_unary_ops.append(floor_opinfo)
 
+frexp_opinfo = OpInfo(
+    clang.frexp,
+    supports_grad=True,
+    dtypes=(datatypes.floating,),
+    sample_input_generator=partial(elementwise_unary_generator, small=True),
+    torch_reference=_elementwise_unary_torch(torch.frexp),
+    test_directives=(
+        DecorateInfo(
+            pytest.mark.skip,
+            "test_phantom_grad_vs_torch_consistency",
+            executors=("nvfuser",),
```

Comment:
What sort of errors are being produced here?  Is this a different problem than was addressed by the change to the meta function?

Reply:
Oops, sorry, I forgot to remove it.